vhdlparser/CMakeLists.txt

```diff
@@ -19,13 +19,27 @@ if (JAVACC_FOUND)
         message(STATUS "  Doxygen requires at least JavaCC version 7.0.5 (installed: ${JAVACC_VERSION})")
 	message(STATUS "  Fall back to JavaCC not installed, using existing files.")
     else()
-
-    add_custom_command(
-        COMMAND ${JAVACC_EXECUTABLE} ${JAVACC_FLAGS} -OUTPUT_DIRECTORY=${PROJECT_SOURCE_DIR}/vhdlparser ${PROJECT_SOURCE_DIR}/vhdlparser/vhdlparser.jj
-        DEPENDS ${PROJECT_SOURCE_DIR}/vhdlparser/vhdlparser.jj
-        OUTPUT ${PROJECT_SOURCE_DIR}/vhdlparser/CharStream.cc ${PROJECT_SOURCE_DIR}/vhdlparser/CharStream.h ${PROJECT_SOURCE_DIR}/vhdlparser/ErrorHandler.h ${PROJECT_SOURCE_DIR}/vhdlparser/ParseException.cc ${PROJECT_SOURCE_DIR}/vhdlparser/ParseException.h ${PROJECT_SOURCE_DIR}/vhdlparser/Token.cc ${PROJECT_SOURCE_DIR}/vhdlparser/Token.h ${PROJECT_SOURCE_DIR}/vhdlparser/TokenManager.h ${PROJECT_SOURCE_DIR}/vhdlparser/TokenMgrError.cc ${PROJECT_SOURCE_DIR}/vhdlparser/TokenMgrError.h ${PROJECT_SOURCE_DIR}/vhdlparser/VhdlParser.cc ${PROJECT_SOURCE_DIR}/vhdlparser/VhdlParser.h ${PROJECT_SOURCE_DIR}/vhdlparser/VhdlParserConstants.h ${PROJECT_SOURCE_DIR}/vhdlparser/VhdlParserTokenManager.cc ${PROJECT_SOURCE_DIR}/vhdlparser/VhdlParserTokenManager.h
-    )
-
+        if (JAVACC_VERSION VERSION_LESS 7.0.6)
```

Comment:
wouldn't it not make more sense to just require JAVACC version 7.0.6+. 
Most people will depend on pre-generated files anyway we already required version 7.0.5+
Then we do not need to maintain a 7.0.5->7.0.5+ upgrade script.

Reply:
That would not be a bad idea, though I didn't want to disrupt toolchains of existing users.